mongo/integration/unified/schema_version.go

```diff
@@ -16,7 +16,7 @@ import (
 
 var (
 	supportedSchemaVersions = map[int]string{
-		1: "1.13",
+		1: "1.15",
```

Comment:
This PR skips all of the 1.14 schema updates, which were added as part of DRIVERS-1671, Go Driver PR #1305 . I'm not entirely sure what the impact of merging this before that ticket would be, likely none, but something to consider.